apps/ern-genturis/src/views/view-public-dashboard.vue

```diff
@@ -110,20 +111,61 @@ import {
   ColumnChart,
   DataTable,
   DataValueHighlights,
+  // @ts-ignore
 } from "molgenis-viz";
 
 import { seqAlongBy } from "../utils/utils";
 import { max } from "d3";
 const d3 = { max };
 
-let loading = ref(true);
-let error = ref(null);
-let registryHighlights = ref({});
-let sexAtBirth = ref({});
-let ageAtInclusion = ref([]);
-let ageAtInclusionTicks = ref([]);
-let enrollmentData = ref({});
-let organisations = ref([]);
+interface IProviderInformation {
+  providerIdentifier: string;
+  hasSubmittedData: boolean;
+}
+
+interface IOrganisations {
```

Comment:
does this match the server? If so, did you know about https://molgenis.github.io/molgenis-emx2/#/molgenis/dev_apps?id=generate-typescript-types-for-an-app (or maybe you used it I just didn't recognize it)